docs/whats-new.asciidoc

```diff
@@ -4,131 +4,91 @@
-
-[role="screenshot"]
-image::whats-new/images/8.9/rule-monitor-dashboard.png[Detection rule monitor dashboard]
+== Detection rules and alerts enhancements
 
 [float]
-=== Automated endpoint response actions for rules
-
-You can now add the {security-guide}/host-isolation-ov.html#isolate-a-host[host isolation response action] to rules. When rule conditions are met, the endpoint is automatically isolated. 
+=== MITRE ATT&CK® coverage page
 
-[float]
-=== Rule exceptions auto-populated with alert data
+The {security-guide}/rules-coverage.html[MITRE ATT&CK® coverage page] shows which MITRE ATT&CK® adversary tactics and techniques are covered by your installed and enabled detection rules. This includes both Elastic prebuilt rules and custom rules.
 
-Now, when you {security-guide}/add-exceptions.html#detection-rule-exceptions[create a new rule exception] from an alert, exception conditions are auto-populated with relevant alert data. A comment describing this action is also automatically added to the *Add comments* section.
+[role="screenshot"]
+image::whats-new/images/8.10/rules-coverage.png[MITRE ATT&CK® coverage page]
 
 [float]
-=== Interactive investigation guides are now generally available
-
-{security-guide}/interactive-investigation-guides.html[Interactive investigation guides], which suggest steps for triaging, analyzing, and responding to potential security issues, are now generally available. You can configure an interactive investigation guide when you create a new rule or edit an existing one.
+=== New prebuilt rule details flyout
 
+The new prebuilt rule details flyout allows you to examine the details of a prebuilt rule before you {security-guide}/prebuilt-rules-management.html[install or update] it. You can access this flyout by clicking a rule name on either the **Add Elastic Rules** page or the **Rule updates** table. The flyout displays the About, Definition, and Schedule sections, as shown on the rule details page. It also shows the setup and investigation guides for rules that have them.
+ 
 [role="screenshot"]
-image::whats-new/images/8.9/IG-UI.png[Interactive investigation guide]
-
+image::whats-new/images/8.10/prebuilt-rule-details-flyout.png[Prebuilt rule details flyout]
 
 [float]
-=== Prebuilt rule updates
-
-Check out the {security-guide}/prebuilt-rules-downloadable-updates.html[latest updates] to prebuilt rules. To download the latest updates, refer to {security-guide}/prebuilt-rules-management.html#update-prebuilt-rules[Update Elastic prebuilt rules]. 
+=== Enhanced alert details flyout UI
 
-[float]
-=== Manage and filter alert tags
+The redesigned alert details experience presents relevant context, insights, and more, while investigating an alert. Use the collapsed view to access summarized information, and then expand each section further to open detailed views. Additional improvements include:
```

Comment:
Nit: This describes expanding it just once, not expanding it and then expanding it again.
```suggestion
The redesigned alert details experience presents relevant context, insights, and more, while investigating an alert. Use the collapsed view to access summarized information, and then expand each section to open detailed views. Additional improvements include:
```